### DISCUSSION TITLE
Add CustomModelData/NBT support for Toast

### DISCUSSION THREAD

**Discussion by mario638** [Zrips/CMI#6965] [Enhancement]
It is posible 
because minecraft vanilla can do it [PHOTO](https://cdn.discordapp.com/attachments/526402563847880725/985120942172823562/unknown.png)
here you can see minecraft vanilla works 
[LINK](https://minecraft.fandom.com/wiki/Advancement/JSON_format)

**Comment by Zrips** [comment]
Will be supported with next CMILib and CMI update

**Comment by mrfloris** [comment]
Then format like `-icon:ironingot{CustomModelData:1234567}` will be able to get used.
